test/lib.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-# Copyright 2019 The Knative Authors
+# Copyright 2020 Google LLC
```

Comment:
if this was a previous file, I think it should keep the 2019, but change to Google LLC... but not entirely sure.

Reply:
Changed it back to `2019`.